x/stake/keeper/key.go

```diff
@@ -104,12 +111,14 @@ func GetDelegationsKey(delegatorAddr sdk.Address, cdc *wire.Codec) []byte {
 
 //________________________________________________________________________________
 
-// get the key for an unbonding delegation
+// get the key for an unbonding delegation by delegator and validator addr.
+// The value at this key is of type stake/types.UnbondingDelegation
 func GetUBDKey(delegatorAddr, validatorAddr sdk.Address, cdc *wire.Codec) []byte {
 	return append(GetUBDsKey(delegatorAddr, cdc), validatorAddr.Bytes()...)
 }
 
 // get the index-key for an unbonding delegation, stored by validator-index
+// The value at this key is a key for the corresponding unbonding delegation.
```

Comment:
As per the spec, these will not actually hold a value soon and simply be used to reconstruct the UBD key if the origin key (this key) is found